Flutter/cartesian-charts/legend.md

```diff
@@ -110,6 +110,59 @@ The [`name`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/char
 
 ![Customized Legend](images/legend/customized_legend.png)
 
+## Customizing legend icon as image
+
+An image is also used as legend icon by using the [`legendItemBuilder`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/Legend/legendItemBuilder.html). If an image is used as legend icon, it should be added in the project and referred as asset in pubspec file.
```

Comment:
already we have explained builder property, I have asked to explain the legendIcon type for image

Reply:
Changed the code snippets and content as we discussed